server/config/application.rb

```diff
@@ -14,5 +14,15 @@ class Application < Rails::Application
     # Settings in config/environments/* take precedence over those specified here.
     # Application configuration should go into files in config/initializers
     # -- all .rb files in that directory are automatically loaded.
+
+    # Allow all CORS requests
+    config.middleware.insert_before 0, Rack::Cors do
+      allow do
+        origins '*'
+        resource '*',
+          headers: :any,
+          methods: %i(delete get head options patch post put)
+      end
+    end
```

Comment:
Could:
 - put this in `server/config/environments/development.rb` instead since we'd only want this in development mode
 - change the origin to `'localhost:*'` or something, although that might mess up local configurations like rails.dev, etc.

Reply:
Yeah -- the `rails.dev` thing definitely weighed on my decision here. I also tried to mirror the settings in the yatrum app (which are in `application.rb` not `development.rb`) so that if this works, that works.

Good thoughts and maybe changes we want to make to both apps if we keep them around longer-term!